app/forms/participants/participant_validation_form.rb

```diff
@@ -0,0 +1,143 @@
+
+    def attributes
+      {
+        step: step,
+        do_you_know_your_trn_choice: do_you_know_your_trn_choice,
+        have_you_changed_your_name_choice: have_you_changed_your_name_choice,
+        updated_record_choice: updated_record_choice,
+        name_not_updated_choice: name_not_updated_choice,
+        trn: trn,
+        name: name,
+        date_of_birth: date_of_birth,
+        national_insurance_number: national_insurance_number,
+        validation_attempts: validation_attempts.to_i, # coerce nil to 0
+      }
+    end
+
+    def date_of_birth=(value)
+      @date_of_birth_invalid = false
+      @date_of_birth = ActiveRecord::Type::Date.new.cast(value)
+    rescue StandardError => _e
+      @date_of_birth_invalid = true
+    end
+
+    def trn_choices
+      [
+        OpenStruct.new(id: "yes", name: "Yes, I know my TRN"),
+        OpenStruct.new(id: "no", name: "No, I do not know my TRN"),
+        OpenStruct.new(id: "i_do_not_have", name: "I do not have a TRN"),
+      ]
+    end
+
+    def name_change_choices
+      [
+        OpenStruct.new(id: "yes", name: "Yes, I changed my name"),
+        OpenStruct.new(id: "no", name: "No, I have the same name"),
+      ]
+    end
+
+    def updated_record_choices
+      [
+        OpenStruct.new(id: "yes", name: "Yes, my name has been updated"),
+        OpenStruct.new(id: "no", name: "No, I need to update my name"),
+        OpenStruct.new(id: "i_do_not_know", name: "I’m not sure"),
+      ]
+    end
+
+    def name_not_updated_choices
+      [
+        OpenStruct.new(id: "register_previous_name", name: "Register for this programme using your previous name (you can update this later)"),
+        OpenStruct.new(id: "update_name", name: "Update your name with the Teaching Regulation Agency"),
+      ]
+    end
+
+    def pretty_date_of_birth
+      if date_of_birth.present?
+        date_of_birth.to_s(:govuk)
+      end
+    end
+
+    def increment_validation_attempts
+      if validation_attempts.nil?
+        @validation_attempts = 1
+      else
+        @validation_attempts += 1
+      end
+    end
+
+  private
+
+    def trn_choice
+      if do_you_know_your_trn_choice.blank? || !do_you_know_your_trn_choice.in?(%w[yes no i_do_not_have])
+        errors.add(:do_you_know_your_trn_choice, :blank)
+      end
+    end
+
+    def name_change_choice
+      if have_you_changed_your_name_choice.blank? || !have_you_changed_your_name_choice.in?(%w[yes no])
+        errors.add(:have_you_changed_your_name_choice, :blank)
+      end
+    end
+
+    def confirm_updated_record_choice
+      if updated_record_choice.blank? || !updated_record_choice.in?(%w[yes no i_do_not_know])
+        errors.add(:updated_record_choice, :blank)
+      end
+    end
+
+    def confirm_name_not_updated_choice
+      if name_not_updated_choice.blank? || !name_not_updated_choice.in?(%w[register_previous_name update_name])
+        errors.add(:name_not_updated_choice, :blank)
+      end
+    end
```

Comment:
It might be a little cleaner to re-use `have_you_changed_your_name_choice.map(&:id)` and so on for the second condition of these validations. Then we needn't repeat the choices since we already know them. One for another PR, though.